### ISSUE TITLE
Have installation scripts fetch go depenencies

### ISSUE THREAD

**Issue by EmperorDali** [czbiohub/dashit#5]
We need to run this during the install:

```
go get github.com/shenwei356/xopen
go get github.com/shenwei356/bio/seq
```

**Comment by Arkadiy-Garber** [comment]
As of November 2019, the installation script does not fetch these dependencies.

causing this error:

```
Processing dependencies for dashit-filter==2.0
Finished processing dependencies for dashit-filter==2.0
cd misc && go build -o score_guides
score_guides.go:6:2: cannot find package "github.com/shenwei356/bio/seq" in any of:
	/usr/local/Cellar/go/1.13.4/libexec/src/github.com/shenwei356/bio/seq (from $GOROOT)
	/Users/arkadiygarber/go/src/github.com/shenwei356/bio/seq (from $GOPATH)
score_guides.go:7:2: cannot find package "github.com/shenwei356/bio/seqio/fastx" in any of:
	/usr/local/Cellar/go/1.13.4/libexec/src/github.com/shenwei356/bio/seqio/fastx (from $GOROOT)
	/Users/arkadiygarber/go/src/github.com/shenwei356/bio/seqio/fastx (from $GOPATH)
score_guides.go:8:2: cannot find package "github.com/shenwei356/xopen" in any of:
	/usr/local/Cellar/go/1.13.4/libexec/src/github.com/shenwei356/xopen (from $GOROOT)
	/Users/arkadiygarber/go/src/github.com/shenwei356/xopen (from $GOPATH)
make: *** [build-score-guides] Error 1

```
Thus, before running make, users must run the following three commands:

`go get github.com/shenwei356/xopen `
`go get github.com/shenwei356/bio/seq `
`go get github.com/shenwei356/bio/seqio/fastx `

Thanks,
Arkadiy

**Comment by EmperorDali** [comment]
This is fixed in ae22f55c9a4a9b4f442ec3d86fd7cdeba4515b7b
